app/views/merchant/orders/show.html.erb

```diff
@@ -0,0 +1,24 @@
+<h1>Merchant Orders</h1>
+
+    <% @orders.each do |order| %>
+      <p><%= order.name %></p>
+      <p><%= order.address %></p>
+        <br/>
+      <h2>Items:</h2>
+        <% order.items_by_merchant(@current_user.merchant.id).each do |item| %>
+        <section id= 'item-<%=item.id%>'>
+          <h5>Item: <%= item.name %></h5>
+          <%= image_tag item.image %>
+          <h5>Price: <%= item.price %></h5>
+          <h5>Quantity: <%= item.order_count %></h5>
+          <% if item.fulfillable?(item.order_count) && item.item_order(order.id).status != "Fulfilled" %>
+          <%= button_to "Fulfill", "/merchant/orders/#{order.id}", params: { item_order_id: item.id }, method: :post %>
```

Comment:
We can improve the readability of this section adding indentations. Other than that, it looks super organized and clean.